src/App.tsx

```diff
@@ -15,37 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  SORT_BY_CHAR = 'SORT_BY_CHAR',
+  SORT_BY_LENGTH = 'SORT_BY_LENGTH',
+}
+
+type TGoods = string[];
+
+interface IGoodsProps {
+  goods: Readonly<TGoods>;
+}
+
+const Goods: FC<IGoodsProps> = ({ goods }) => (
+  <ul>
+    {goods.map(el => (
+      <li data-cy="Good" key={el}>
+        {el}
+      </li>
+    ))}
+  </ul>
+);
+
+type TSortGoods = (
```

Comment:
why did u use `interface` above, and `type` here?
Never mix style ur code. Pick something 1 and use it

Reply:
I think there is no difference and for me using types for functions is more understandable and readable

Reply:
in ur case, the `interface` or `type` doesn't give u any bonuses, so I don't see the point in mixing. 
In my opinion, on the contrary, it is not readable, cuz ur code is mixed

If u want to use `interface` for components and `type` for everything else, then do so in all cases in the future.

but I wouldn't do that) 😄